autorest/retriablerequest_1.x.go

```diff
@@ -0,0 +1,39 @@
+// +build !go1.8
```

Comment:
I'm having trouble interpreting this in conjunction with the [build constraints documentation](https://golang.org/pkg/go/build/). Will this function as <1.8 or <1.8>? i.e. when 1.9 gets released, will this get built?